src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.iOS/Airship.cs

```diff
@@ -111,6 +113,25 @@ public string NamedUser
             }
         }
 
+        private InboxHandler onMessageCenterUpdated;
+        public event InboxHandler OnMessageCenterUpdated
+        {
+            add
+            {
+                onMessageCenterUpdated += value;
+                NSNotificationCenter.DefaultCenter.AddObserver(aName: (NSString)"com.urbanairship.notification.message_list_updated", (notification) =>
+                {
+                    Console.WriteLine("Listener Set");
```

Comment:
We should remove this line. We probably need a Logger class but that can be done in another ticket.